.github/workflows/ci.yml

```diff
@@ -13,125 +13,145 @@ jobs:
             ARCH: x86_64
             MINIFORGE_NAME: "Miniforge3"
             OS_NAME: "Windows"
+            UPLOAD_ARTEFACT: true
```

Comment:
Let's remove these and upload artifacts always

Reply:
Yes, I was annoyed with getting one large artefact but setting the name of the artefact as you suggested above fix it!